### PR TITLE
Fix python3 support. May break python2.

### LINKED ISSUE #1640: Can not find python3
Hello again,

I have a python3 codebase with pip dependencies.

When I run

```
docker/run.sh "-v /my-project:/project" --info analyze -f JSON -i /project -o /project/ort/analyzer
```

I get back a json with:
```
        "PIP::requirements.txt:" : [ {
          "timestamp" : "2019-07-12T16:05:36.880494Z",
          "source" : "PIP",
          "message" : "IOException: Cannot run program \"python3\": error=2, No such file or directory\nCaused by: IOException: error=2, No such file or directory",
          "severity" : "ERROR"
        } ]
```

Best,
Felix

### PR DESCRIPTION
**Pull request by flixx**
Fixes #1640 

However this may break python2 support.
Before merging it should be tested if python2 projects still work fine.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1642)

### CODE REVIEW

**Comment by sschuberth** [comment]
I was a few minutes quicker, see https://github.com/heremaps/oss-review-toolkit/pull/1641 😋 But anyway, I'd rather have your contribution than mine, I've contributed enough already 😉

File: docker/run/Dockerfile [hunk 1]
```diff
@@ -50,6 +50,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         # Install general tools required by this Dockefile.
         curl \
+        python3 \
```
Comment (sschuberth) [inline docker/run/Dockerfile]: I think we can and should drop this as it's a dependency of `python3-pip` already (and we don't install Python 2 here either).

File: docker/run/Dockerfile [hunk 2]
```diff
@@ -74,7 +76,7 @@ RUN \
     chmod a+x /usr/local/bin/repo && \
     # Install package managers (in versions known to work).
     npm install --global bower@$BOWER_VERSION yarn@$YARN_VERSION && \
-    pip install virtualenv==$PYTHON_VIRTUALENV_VERSION && \
+    pip3 install virtualenv==$PYTHON_VIRTUALENV_VERSION && \
```
Comment (sschuberth) [inline docker/run/Dockerfile]: Is this change really required for your issue to get fixed?

**Comment by flixx** [comment]
Also just saw it ;).

That is nice of you, but I also tested your PR and that also works.
So go ahead and merge yours since it will not break python2...

**Comment by sschuberth** [comment]
Ok, so closing this in favor of https://github.com/heremaps/oss-review-toolkit/pull/1641 (which will still need a peer-review before I can merge it).
